### DISCUSSION TITLE
Transcribe Flask Application

### DISCUSSION THREAD

**Discussion by hussamsayeed** [openai/whisper#596] [Show and tell]
I have been trying to write the flask application. but for the audio input it is taking as file. Error Attached!
Could anyone help me with this for building the web app. which could be used as api?

![Screenshot (808)](https://user-images.githubusercontent.com/46478199/204077168-694a7176-e72a-4a5c-ab0b-943aa88f4deb.png)

**Comment by Kabanosk** [comment]
You can try FastAPI it is very similar to Flask in building applications and it has easier file operating in my opinion 😁
